llvm/test/Transforms/CheriBoundAllocas/multiple-uses-in-same-instr.ll

```diff
@@ -13,11 +13,11 @@ target triple = "riscv64-unknown-freebsd13"
 %struct.QTypedArrayData = type { %struct.QArrayData }
```

Comment:
Fold this commit into the original test case one?

Reply:
Any further suggestions for this PR? If not I'll address this comment and merge then.